packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -281,7 +281,7 @@ async def _receive(self) -> Optional[bytes]:
             if not data:  # pragma: no cover
                 return None
             return data
-        except asyncio.streams.IncompleteReadError as e:
+        except asyncio.streams.IncompleteReadError as e:  # pragma: no cover
```

Comment:
on my local machine, with file (missing from develop) `test_errors.py`, this is already covered by  `tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py::TestLibp2pClientConnectionNodeDisconnected::test_node_disconnected`
(https://github.com/fetchai/agents-aea/blob/feature/libp2p-improve-coverage/tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py#L128)